teps/0092-scheduling-timeout.md

```diff
@@ -46,13 +46,14 @@ authors:
 ## Summary
 
 Add runtime configuration options to allow setting a timeout for `TaskRuns` which applies before the `TaskRun's`
-underlying pod is scheduled, and only start our existing timeout counters after this timeout has passed, if specified.
-
+underlying pod is scheduled (Scheduling), and only start our existing timeout counters (Execution) after this timeout has passed, if specified.
+Add a Total Timeout which will can be set if users don't care about the distribution between scheduling and execution timeouts.
 ## Motivation
 
 Current timeout logic doesn't take into account the time a pod may spend waiting to be scheduled; the timeout countdown
 starts as soon as the TaskRun starts. In some environments the pod backing the TaskRun may not start executing until
-some period of time after that.
+some period of time after that. In order to keep conistent timeout definitions across pipelines, if the timeout is
```

Comment:
nit: keeping timeout definitions consistent between CRDs is more of a goal of the proposal than a motivation for it. There's also two components to consistency: starting at start time vs creation time, and the meaning of 0 timeout.